pipelines/package/darwin_arm.yml

```diff
@@ -0,0 +1,19 @@
+steps:
+- task: DownloadSecureFile@1
+  inputs:
+    secureFile: 'NORDIC_SIGNING_CERTIFICATE.p12'
+  displayName: 'Download certificate'
+- bash: |
+    rm -rf release
```

Comment:
Why is this needed here but not when packaging releases on the other platforms?

Reply:
It uses our internal build server that doesn't clean everytime, I'm afraid previous releases will be lying around in the release folder and get picked up by the copy * command